src/app/[locale]/(large-layout)/northstar/page.tsx

```diff
@@ -3,10 +3,16 @@ import { getServerTranslation } from '@/helpers/getServerTranslation'
 import { getMetadataObject } from '@/helpers/metadata/getMetadataObject'
 import NorthStarIframe from './_components/NorthStarIframe'
 
-export async function generateMetadata() {
-  const { t } = await getServerTranslation()
+export async function generateMetadata({
+  params,
+}: {
+  params: Promise<{ locale: string }>
+}) {
+  const { locale } = await params
+  const { t } = await getServerTranslation(locale)
```

Comment:
Why not pass params directly to the `getServerTranslation` ? => less diff

Reply:
je l'ai fait là ou c'était possible de le faire ; `getMetadataObject` a besoin de `locale` ça limite pas mal l'utilité de cette modif